sdk/communication/azure-communication-sms/src/test/java/com/azure/communication/sms/SmsLiveAsyncClientTests.java

```diff
@@ -76,7 +76,7 @@ public void sendSmsRequestAsyncSingleNumberNoDeliveryReport(HttpClient httpClien
     }    
 
     private SmsAsyncClient getTestSmsClient(HttpClient httpClient) {
-        return getSmsClientBuilder(httpClient)
+        return getSmsClientBuilderWithConnectionString(httpClient)
```

Comment:
How do we convey that using connectionstring is the way to go for testing? seems like nothing stops the developers from using the builder without connectionString, just something to keep in mind and think about..